fast_api_als/routers/lead_conversion.py

```diff
@@ -42,23 +53,30 @@ def get_quicksight_data(lead_uuid, item):
 
 @router.post("/conversion")
 async def submit(file: Request, token: str = Depends(get_token)):
+    start = time.process_time()
     body = await file.body()
     body = json.loads(str(body, 'utf-8'))
 
     if 'lead_uuid' not in body or 'converted' not in body:
         # throw proper HTTPException
-        pass
+        logging.error("conversion: fn submit  : 'lead_uuid' key or 'converted' key is not present in body")
+        raise HTTPException(status_code = 500, detail="conversion: fn submit : 'lead_uuid' key or 'converted' key is not present in body")
+        
         
     lead_uuid = body['lead_uuid']
     converted = body['converted']
 
     oem, role = get_user_role(token)
     if role != "OEM":
         # throw proper HTTPException
-        pass
-
+        logging.error("conversion: fn submit  : Unauthorized access(role != `OEM`)")
```

Comment:
Not a good practice to write /endpoint and function name in your every log statement.
It's almost taking half of your important log message.

Reply:
Updated the format of log message